### ISSUE TITLE
why your Retinanet performs better than paper (about +3.4mAP) ?

### ISSUE THREAD

**Issue by yangJirui** [TuSimple/simpledet#131]
Hi, I noticed that your RetinaNet get 37.4 mAP with 640*640 img.
About 3.4 mAP higher than paper (with 600*1000 img), 
About 3.8 mAP higher than maskrcnn_benchmark.

Why your Retinanet performs so well ？ 
I guess there are several reasons for this：

1. Your backbone is R50v1b, not resnet in caffe;
2. Training with Multi-scale in your code,  scale_min = 0.8, scale_max = 1.2.
3. 2x training.(25 epoch)

Are the reasons I analyzed correct?
If there are other reasons, can you disclose them?
And what do you think is the most important reason?

Thank you so much.

**Comment by huangzehao** [comment]
Yes. 
This retinanet implementation is following nasfpn paper. Please refer the paper for more details.

**Comment by johnlanbor** [comment]
so, which get the most gain?
1. multi scale train between[0.8, 1.2]
2. 2X training
3. use batchnorm layer after all conv layer
4. 0.08 lr
